src/vma/sock/sock-redirect.cpp

```diff
@@ -2107,6 +2107,12 @@ pid_t fork(void)
 
 		safe_mce_sys().get_env_params();
 		vlog_start("VMA", safe_mce_sys().log_level, safe_mce_sys().log_filename, safe_mce_sys().log_details, safe_mce_sys().log_colors);
+#ifdef DEFINED_RDMA_LIB_RESET
+		if (rdma_lib_reset()) {
```

Comment:
I considered it carefully before suggesting rdma_cma.h and I think functionality originating in librdmacm should go in a rdma correlated files, while libibverbs functionality should go in verbs correlated files.